src/main/java/org/folio/search/service/AsyncFolioExecutionContext.java

```diff
@@ -9,7 +9,7 @@
 import org.folio.spring.FolioModuleMetadata;
 
 @RequiredArgsConstructor
-final class AsyncFolioExecutionContext implements FolioExecutionContext {
+public final class AsyncFolioExecutionContext implements FolioExecutionContext {
```

Comment:
Looks like there will be a same class `AsyncFolioExecutionContext.java `in the FOLIO Spring base app.

Reply:
I've updated version to the newest 1.0.1, but I don't see it in the project, so I leave it for now, when it will be implemented, I will replace it with version from folio-spring-base

Reply:
That PR was closed by @bohdan-suprun this morning. :) https://github.com/folio-org/folio-spring-base/pull/13